conu/apidefs/backend.py

```diff
@@ -0,0 +1,53 @@
+"""
+Definition for a backend class
+"""
+
+import logging
+
+from .container import Container
+from .image import Image
```

Comment:
Is there any reason, why you use relative path? I guess, `from conu.apidefs.container` is better from readability point of view.

Reply:
I guess it's a matter of personal taste, I don't mind changing all to absolute -- my version is shorter and in my opinion obvious; what do others think? @dhodovsk @jscotka @bkabrda